Source/StreetMapRuntime/Public/StreetMap.h

```diff
@@ -50,6 +50,12 @@ struct STREETMAPRUNTIME_API FStreetMapMeshBuildSettings
 	UPROPERTY(Category = StreetMap, EditAnywhere, DisplayName = "Create 3D Buildings")
 	uint32 bWant3DBuildings : 1;
 
+	/** building level floor conversion factor in centimeters
+		@todo: harmonize with OSMToCentimetersScaleFactor refactoring
+	*/
+	UPROPERTY(Category = StreetMap, EditAnywhere, DisplayName = "Building Level Floor Factor")
+	float bBuildingLevelFloorFactor = 300.0f;
```

Comment:
Should not have a 'b' prefix, per UE4 coding conventions, that is only used for bools.  

Also, initializing members inline like this is not something that we've done in the engine before, but I think it should be supported on compilers and work with UObjects.  So it's cool. :)